internal/scheduler/loadbalancealgoimpl.go

```diff
@@ -49,6 +51,14 @@ func (algo *LoadBalanceAlgoImpl) GetNode(nodebindings []*nodebinding.Model, node
 		})
 	}
 
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Let's consider moving this to a util package once we have a fair amount of confidence in generics. Preferably with 1.20.

Reply:
@sakshi-goyal-razorpay since we are using the same piece of code here and in schedulerTask file, can we consider reusing the same code block.

Reply:
I tried that, but without the use of generics, it is not possible, since the array type is different